src/app/components/FrostedGlassPromo/FrostedGlassPanel.tsx

```diff
@@ -0,0 +1,59 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx, useTheme } from '@emotion/react';
+import { PropsWithChildren } from 'react';
+
+import useImageColour from '../../hooks/useImageColour';
+
+import styles from './styles';
+
+const BLUR_RADIUS = 15;
+
+type FrostedGlassPanelProps = {
+  image: string;
+  minimumContrast?: number;
+  paletteSize?: number;
+};
+
+const FrostedGlassPanel = ({
+  image,
+  children,
+  minimumContrast = 8,
+  paletteSize = 10,
+}: PropsWithChildren<FrostedGlassPanelProps>) => {
+  const {
+    palette: { GREY_8 },
```

Comment:
Can the styling here and within the rest of the component file be extracted out into the co-located `style` file or does it not make sense to do so?

Reply:
The `GREY_8` in this instance is used for the `useImageColour` hook below, so it will need passed in separately.

The styles are a bit of mixture here, both the `style.ts` file and inline styling since we need to vary the styles based on certain props. Passing props to `css()` isn't the same as the older `styled` components, so requires inlining them. I think Emotion creates classes as before though, so its not actually inlining styles in the typical CSS way.